demo/website/src/components/chat/components/ExportChat.tsx

```diff
@@ -100,13 +118,31 @@ export default function ExportChat(props: { chat: Chat }) {
         >
           <SpaceBetween size="m">
             <FormField label="Export format" stretch={true}>
-              <SegmentedControl
-                selectedId={exportFormat}
-                onChange={({ detail }) => {
-                  setExportFormat(detail.selectedId as ExportFormat);
-                }}
-                options={exportFormatOptions}
-              />
+              <SpaceBetween size="m" direction="horizontal">
+                <SegmentedControl
+                  selectedId={exportFormat}
+                  onChange={({ detail }) => {
+                    setExportFormat(detail.selectedId as ExportFormat);
+                  }}
+                  options={exportFormatOptions}
+                />
+                <Checkbox
+                  checked={includeModelKwargs}
+                  onChange={({ detail }) => {
+                    setIncludeModelKwargs(detail.checked);
+                  }}
+                >
+                  Include model kwargs
+                </Checkbox>
+                <Checkbox
+                  checked={includeLLMInfo}
+                  onChange={({ detail }) => {
+                    setIncludeLLMInfo(detail.checked);
+                  }}
+                >
+                  Include LLM info
+                </Checkbox>
```

Comment:
We should callout that these are only available for "local requests", not persisted messages. Otherwise it seems like broken if you refresh and their are no details for these fields... or for old messages in chat not having these details.